Docker 2/Docker 2.txt

```diff
@@ -0,0 +1,4 @@
+1. docker load -i hello-world.tar
+2. docker run hello-world:latest
+3. docker start 3ab138d9d3bc
```

Comment:
no need to use docker run and docker start, "start" is for existing containers, "run" also creates them.